control-plane/grpc/proto/v1/replica/replica.proto

```diff
@@ -71,7 +60,7 @@ message CreateReplicaRequest{
   // size of the replica in bytes
   uint64 size = 6;
   // protocol to expose the replica over
-  Protocol share = 7;
+  common.Protocol share = 7;
```

Comment:
maybe replicas need their own protocol, they can only be shared by Nvmf.

Reply:
Its there, The ReplicaShareProtocol enum

Reply:
I have used it according to its type in message_bus. https://github.com/openebs/mayastor-control-plane/blob/263a8222fb9fe632cb5a977df3f38f1ca95dbfb6/common/src/types/v0/message_bus/replica.rs#L150

Reply:
Ah so changing it will break things? Ok we can do it on another PR